### DISCUSSION TITLE
increase version of solaredge_modbus

### DISCUSSION THREAD

**Discussion by JanJaapKo** [addiejanssen/domoticz-solaredge-modbustcp-plugin#72] [General]
solaredge_modbus version 0.7.0 doesn't work anymore (at least not on new install), current version is 0.8.0.

**Comment by addiejanssen** [comment]
Did you test the plugin with solaredge_modbus 0.8.0? Is it working?
If so: which version of the domoticz-solaredge-modbustcp-plugin did you use? V1 or V2?

**Reply by JanJaapKo** [reply]
I guess I used V1 (was not even aware of v2 honestly), just did:
`git clone https://github.com/addiejanssen/domoticz-solaredge-modbustcp-plugin`

And yes, I have solaredge_modbus 0.8.0 which works like a charme!

**Comment by bamloeff** [comment]
Soloredge_modbus 0.7.0 works for me if I replace pymodbus 3.5.4 for 2.4.0.
